tests/data/cc/dataset/auxillary_datasets/maintenances/Maintenance updates.json

```diff
@@ -0,0 +1,79 @@
+{
```

Comment:
The filename of this dataset is quite different than of the other datasets in the tests (no snake case, space, ...).